lib/minitest/parallel_fork.rb

```diff
@@ -23,9 +23,9 @@ def self.on_parallel_fork_marshal_failure(&block)
   module Unparallelize
     define_method(:run_one_method, &Minitest::Test.method(:run_one_method))
   end
-  
-  def self.parallel_fork_stat_reporter(reporter)
-    reporter.reporters.detect do |rep|
+
+  def self.parallel_fork_stat_reporters(reporter)
```

Comment:
This would break the public API, as this method was specifically added to be overridden (see #3).  Please remove the changes to this method.

Reply:
Ah, ok, I missed that part.  The rest, of course, follows.  Thanks!